x-pack/legacy/plugins/siem/public/components/inspect/modal.tsx

```diff
@@ -49,7 +104,57 @@ export const ModalInspectQuery = ({
   if (!isShowing || request == null || response == null) {
     return null;
   }
+  const inspectRequest: Request | null = parseRequest(request);
+  const inspectResponse: Response | null = parseResponse(response);
+  const statistics: Array<{
+    title: NonNullable<ReactNode | string>;
+    description: NonNullable<ReactNode | string>;
+  }> = [
+    {
+      title: (
+        <>
+          <EuiIconTip content={i18n.INDEX_PATTERN_DESC} position="top" />
```

Comment:
Also, I'm of the mindset to begin changing many (or all) of our `questionInCircle` icon tooltips to instead use the `iInCircle` icon. The reason for this is because the use cases we typically have are providing the user with a short snippet of information and not leading them toward any more robust documentation or instruction.

That said, can we add a prop of `type="iInCircle"` here?